parsl/monitoring/db_logger.py

```diff
@@ -30,23 +32,90 @@ def emit(self, record):
         pass
 
 
-def get_db_logger(host='search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-east-1.es.amazonaws.com',
-                  port=443,
-                  enable_es_logging=False,
-                  index_name="my_python_index",
-                  version='1.0.0',
+class LoggerConfig():
+    """ This is a config class for creating a logger. """
+    def __init__(self,
+                 host=None,
+                 port=None,
+                 enable_ssl=True,
+                 logger_type='local_database',
+                 index_name="my_python_index",
+                 logger_name='parsl_db_logger',
+                 eng_link=None,
+                 version='1.0.0',
+                 web_app_host='http://localhost',
+                 web_app_port=8899,
+                 resource_loop_sleep_duration=15,
+                 workflow_name=None):
+        """ Initializes a db logger configuration class.
+
+        Parameters
+        ----------
+        host : str
+            Used with Elasticsearch logging, the location of where to access Elasticsearch. Required when using logging_type = 'elasticsearch'.
+        port : int
+            Used with Elasticsearch logging, the port of where to access Elasticsearch. Required when using logging_type = 'elasticsearch'.
+        enable_ssl : Bool, optional
+            Used with Elasticsearch logging, whether to use ssl when connecting to Elasticsearch.
+        logger_type : str, optional
+            Determines whether to use Elasticsearch logging or local database logging, defaults to 'local_database' and accepts 'elasticsearch'.
+        index_name : str, optional
+            Used with Elasticsearch logging, the name of the index to log to.
+        logger_name : str, optional
+            Used with both Elasticsearch and local db logging to define naming conventions for loggers.
+        eng_link : str, optional
+            Used with local database logging, SQLalchemy engine link to define where to connect to the database. If not set, DFK init will use a sqlite3
+            database inside the rundir.
+        version : str, optional
+            Optional workflow identification to distinguish between workflows with the same name, not used internally only for display to user.
```

Comment:
This can be done with the uuid run_id that is unique to each workflow. I believe the suggestion for having a simple version was to potentially give users something easier to work with that they could control.